api/src/main/java/io/minio/messages/Expiration.java

```diff
@@ -45,13 +50,26 @@ public Expiration(
     }
 
     this.expiredObjectDeleteMarker = expiredObjectDeleteMarker;
+    this.expiredObjectAllVersions = expiredObjectAllVersions;
   }
 
-  public Expiration(ZonedDateTime date, Integer days, Boolean expiredObjectDeleteMarker) {
-    this(date == null ? null : new ResponseDate(date), days, expiredObjectDeleteMarker);
+  public Expiration(
+      ZonedDateTime date,
+      Integer days,
+      Boolean expiredObjectDeleteMarker,
+      Boolean expiredObjectAllVersions) {
+    this(
+        date == null ? null : new ResponseDate(date),
+        days,
+        expiredObjectDeleteMarker,
+        expiredObjectAllVersions);
   }
 
   public Boolean expiredObjectDeleteMarker() {
     return expiredObjectDeleteMarker;
   }
+
+  public Boolean expiredObjectAllVersions() {
```

Comment:
Add comment like https://github.com/minio/minio-java/blob/master/api/src/main/java/io/minio/messages/Item.java#L112

Reply:
This is minio extension. Refer the comment line I shared and have it like that.